registry.json

```diff
@@ -1,4 +1,19 @@
 [
+   {
+      "ticker":"ONE1",
+      "poolId":"pool1n2yp66zk6m33sg8k20hsvmk9tw7xeeswzdcfxhnx9hm9jkxv5k4",
+      "social":{
+         "facebook":"",
+         "twitter":"@ADAOnePool",
+         "website":"https://adapool.one",
+         "reddit":"",
+         "instagram":"",
+         "youtube":"https://www.youtube.com/channel/UCaPUgQuIwYmQHea12Yen2ug"
+      },
+      "telegram":"https://t.me/joinchat/XVIgiKelvKFhOGQ5",
```

Comment:
You need to provide a Telegram usermame is this field, not a group.  Your telegram account will be used for SPA voting purposes.

Reply:
```suggestion
      "telegram":"@ADAONEPool",
```